### PR TITLE
Add `osu!taiko Malaysian Tournament 2022`

### PR DESCRIPTION
**Pull request by cdhsausageboy**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Review by Walavouchey — CHANGES_REQUESTED** [review]
- replace `’` with `'`
- don't capitalise "tiebreaker" in the middle of sentences

will check more thoroughly a bit later

**Comment by Walavouchey** [comment]
image


<https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits>

image

we'll squash the commits for you before merging (hence `state:needs rebase` label), but in the future, try to condense the amount of commits. this can easily be done on the github website by creating a new branch and copying the final changes to it.

File: wiki/Tournaments/OTMT/2022/en.md [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=MY }:: [CrabCow](https://osu.ppy.sh/users/9755504) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=MY }:: [HHVanilla Ice](https://osu.ppy.sh/users/12803930) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=MY }:: [Jyakeii](https://osu.ppy.sh/users/12086753) |
+| *4th place* | ::{ flag=MY }:: [vernonlim](https://osu.ppy.sh/users/10167542) |
+
+## Mappools
+
+🎵 **= Custom song**
```
[the remark this answers is not in the document]
Comment (cdhsausageboy) [reply wiki/Tournaments/OTMT/2022/en.md:96]: done.

File: wiki/Tournaments/OTMT/en.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# osu!taiko Malaysian Tournament
+
+Index page for all iterations of the osu!taiko Malaysian Tournament (OTMT).
+
+- [OTMT 2022](2022)
```
[the remark this answers is not in the document]
Comment (cdhsausageboy) [reply wiki/Tournaments/OTMT/en.md:5]: done.

File: wiki/Tournaments/OTMT/2022/en.md [hunk 3]
```diff
@@ -0,0 +1,469 @@
...
+| --: | :-- |
+| Registration phase | 2022-09-17 to 2022-10-01 |
+| Qualifiers Showcase | 2022-10-08 |
+| Qualifiers & Seedings Reveal | 2022-10-15 to 2022-10-16 |
+| Group stage | 2022-10-22 to 2022-10-23 |
+| Round of 16 | 2022-10-29 to 2022-10-30 |
+| Quarterfinals | 2022-11-05 to 2022-11-06 |
+| Semifinals | 2022-11-12 to 2022-11-13 |
+| Finals | 2022-11-19 to 2022-11-20 |
+| Grand Finals | 2022-11-27 |
```
[the remark this answers is not in the document]
Comment (cdhsausageboy) [reply wiki/Tournaments/OTMT/2022/en.md:25]: done.

File: wiki/Tournaments/OTMT/2022/en.md [hunk 4]
```diff
@@ -0,0 +1,469 @@
...
+
+## Prizes
+
+| Placing | Prize |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | RM407 (~94USD), 4 month osu Supporter Tag, Custom Deskpad, Custom IRIS Voice Line, Profile Badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | RM207 (~48USD), 2 month osu Supporter Tag, Custom Deskpad, Custom IRIS Voice Line |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | RM107 (~25USD), 1 month osu Supporter Tag, Custom Deskpad, Custom IRIS Voice Line |
+| *4th place* | RM56 (~13USD), 1 month osu Supporter Tag, Custom IRIS Voice Line |
```
[the remark this answers is not in the document]
Comment (cdhsausageboy) [reply wiki/Tournaments/OTMT/2022/en.md:34]: changed the first two points. custom iris voice line refers to a victory voice line customised to the winner of the tournament narrated by our mascot iris.

**Comment by Walavouchey** [comment]
squashed 100+ commits

**Review by Walavouchey — DISMISSED** [review]

**Review by Walavouchey — DISMISSED** [review]

**Review by Walavouchey — DISMISSED** [review]

**Comment by Walavouchey** [comment]
please check my commits to see what changes were required. in particular, make sure to follow the [tournament template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md)

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**